### DISCUSSION TITLE
Implement Popular Product Slider Using Breeze Slider

### DISCUSSION THREAD

**Discussion by JamesDenish** [breezefront/community#185] [1. Help] [answered]
I want to add slider for products on homepage. Needs to implement in draggable form.

Any one knows kindly help me out.

**Comment by vovayatsyuk** [comment]
Use the built-in Magento product list widget:

![image](https://github.com/breezefront/community/assets/306080/97ac98e3-e9cd-48a0-8fa1-2d501359bfb0)

**Reply by JamesDenish** [reply]
But for related how I can add pagebuilderSlider to ol li structure I can't able to use it in proper manner.

Can you please guide me.

**Reply by JamesDenish** [reply]
![image](https://github.com/breezefront/community/assets/142228975/7355fb63-c949-4719-87a9-e6b00ecc8acb)
This is ol >li  structure where I want pagebuilderSlider() is implied but not working.

And provide limits as well to show elment in one slide view and buttons yes or no.

So please suggest me

**Reply by vovayatsyuk** [reply, marked as answer]
You can add `breeze-carousel` CSS class around listing: https://github.com/orgs/breezefront/discussions/35#discussioncomment-4423627 and breeze will init carousel automatically: https://github.com/breezefront/theme-frontend-breeze-blank/blob/master/web/js/breeze/theme-blank.js#L63-L68

Or, you can init slider manually:

```js
$('.block.upsell .products-grid').pagebuilderCarousel({
    arrows: false,
    dots: true
});
```
